<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>877</width>
    <height>517</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(216, 240, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>481</width>
      <height>431</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(201, 232, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcdNumber_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(154, 154, 154);
</string>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
    </widget>
    <widget class="QSlider" name="Joint_Slider_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Joint 5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Joint 3</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(154, 154, 154);
</string>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
    </widget>
    <widget class="QSlider" name="Joint_Slider_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="Joint_Slider_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="Joint_Slider_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(154, 154, 154);
</string>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Joint 1</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(154, 154, 154);
</string>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
    </widget>
    <widget class="QSlider" name="Joint_Slider_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Joint 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Joint 4</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(154, 154, 154);
</string>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(201, 232, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="px">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(179, 208, 227);</string>
     </property>
     <property name="text">
      <string>+ X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ny">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(179, 208, 227);</string>
     </property>
     <property name="text">
      <string>-Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nx">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(179, 208, 227);</string>
     </property>
     <property name="text">
      <string>- X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="py">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(179, 208, 227);</string>
     </property>
     <property name="text">
      <string>+ Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pz">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(179, 208, 227);</string>
     </property>
     <property name="text">
      <string>+ Z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nz">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(179, 208, 227);</string>
     </property>
     <property name="text">
      <string>- Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(201, 232, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="z_coordinate">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(154, 154, 154);
</string>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="x_coordinate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(154, 154, 154);
</string>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
    </widget>
    <widget class="QPushButton" name="homeconfig">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Home Configuration</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="y_coordinate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(154, 154, 154);
</string>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="visioncontrol">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vision Control</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Joint_Slider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Joint_Slider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Joint_Slider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Joint_Slider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Joint_Slider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
